### DISCUSSION TITLE
Discord, phone number, and IRC

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [hyprwm/Hyprland#1307] [Q&A]
I don't want to give Discord my phone number, but I cannot DM you:
![image](https://user-images.githubusercontent.com/54762742/210221684-93b36d0d-c396-414a-bc24-122417510ede.png)

Can we have a Libera IRC channel bridged to the Discord server?

**Comment by vaxerski** [comment]
my dms are open and people DM me every now and then with requests for me to skip the verification for them, no clue why it won't work for you. You could always send a friend request.

No IRC will be made for Hyprland.

**Comment by raffaelemancuso** [comment]
> No IRC will be made for Hyprland.

why?

**Reply by vaxerski** [reply]
No member from the moderation team has the will to moderate the other side. Furthermore, discord provides nifty features for automatically filtering out stuff like spam or explicit content, making our job easier.

**Comment by TheSunCat** [comment]
I would also really appreciate a bridge between the Discord server and a more open platform, such as Matrix or IRC.

**Reply by NotAShelf** [reply]
I don't think it's happening, chief. It has been brought up multiple times and the answer has been no everytime.

**Reply by TheSunCat** [reply]
I know, but I will voice my opinion regardless as Hyprland is pretty much the only reason I still have my Discord account. If the issue is moderation, I am sure some community members will be happy to step up if it means independence from Discord as a platform.

**Reply by NotAShelf** [reply]
Community members are more or less free to create their own unofficial Hyprland communities on those platforms (like how there are unofficial Archlinus and NixOS servers) but all support given there would be unofficial as I don't see any of the moderation team looking to migrate to matrix (or especially fucking IRC, this is not 2015). Bridges would also not exist, because once again, no interest from us. Not to mention giving some degree of authority to people who may go rogue.

**Reply by TheSunCat** [reply]
Fair. And yeah I would not go for IRC at all haha, just mentioning it since raffaem did. I wonder if there's any bridged moderation possible, so moderators can handle Matrix from within Discord?

**Reply by vaxerski** [reply]
there is no. Furthermore, there are no content filters on matrix afaik

**Reply by TheSunCat** [reply]
Matrix has an excellent moderation tool called [mjolnir](https://github.com/matrix-org/mjolnir) :)

**Reply by raffaelemancuso** [reply]
> Fair. And yeah I would not go for IRC at all haha, just mentioning it since raffaem did.

![team_chat](https://user-images.githubusercontent.com/54762742/210898591-860154f3-c12e-4794-8431-666dc119a330.png)

**Reply by vaxerski** [reply]
you're basically posting a meme making fun of yourself

we're slowly working with the company I work at to get a privacy-respecting alternative up and running, we will bridge there when it will enter a public beta.

IRC or matrix are most likely not happening, unless of course, someone wants to make an unofficial one. I will not send a cease and desist letter. cba.

**Reply by raffaelemancuso** [reply]
Yes

> we're slowly working with the company I work at to get a privacy-respecting alternative up and running

You mean you are working on a protocol alternative to Matrix? Why?

> we will bridge there when it will enter a public beta.

![image](https://user-images.githubusercontent.com/54762742/210970739-2f4fab04-6383-49de-bdaa-0483097a4452.png)

> IRC or matrix are most likely not happening, unless of course, someone wants to make an unofficial one. I will not send a cease and desist letter. cba.

I still have to find a good client for Matrix. They are all bugged / don't work for me. (what does cba stands for?)

**Reply by TheSunCat** [reply]
[nheko](https://github.com/Nheko-Reborn/nheko) runs pretty well on Hyprland and is a native program (no embedded browser like Element). It does tend to crash sometimes, but if you don't use the system theme option is looks and works quite great. Right-click menus are borked on Wayland due to its use of Qt5, but this is likely going to change soon.

**Reply by raffaelemancuso** [reply]
> Right-click menus are borked on Wayland due to its use of Qt5, but this is likely going to change soon.

Yes I have noted this on different softwares. Why is that? Do you have any references and what is the software that needs patching?

I'm running fractal-git now. The stable version doesn't login.

**Reply by TheSunCat** [reply]
Qt5's wayland support is very buggy in my experience. I guess you could try patching the system install of qt5, but so far I have had to just deal with the bugs and wait for apps to update.

**Reply by NovaAndrom3da** [reply]
Since I've seen it requested multiple times, I'll set up a community matrix space for Hyprland.

**Reply by NovaAndrom3da** [reply]
https://github.com/hyprwm/Hyprland/discussions/1891

**Comment by raffaelemancuso** [comment]


**Reply by vaxerski** [reply]
ummm wrong thread?

**Reply by raffaelemancuso** [reply]
thanks, moved upside
